src/ofxstatement/tests/test_plugin.py

```diff
@@ -1,6 +1,9 @@
 import unittest
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
```

Comment:
We don't support the ancient versions of Python, so I think this precaution is unnecessary (unless I'm missing something).